src/styles/blocks/card.scss

```
@@ -0,0 +1,100 @@
+* {
+  margin: 0;
+  font-family: "Roboto", serif;
+  box-sizing: border-box;
+}
+
+.card {
+  width: 200px;
+  display: flex;
+  flex-direction: column;
+  border: 1px solid rgb(243, 243, 243);
+  border-radius: 5px;
+  background-color: white;
+  padding-inline: 16px;
```

try to change your padding
```suggestion
  padding: 16px;
```